servers/sklearnserver/sklearnserver/requirements.txt

```diff
@@ -1,4 +1,4 @@
 seldon_core
-scikit-learn == 0.24.2
-numpy >= 1.8.2
+scikit-learn == 0.23.0
```

Comment:
Do you know why the downgrade of both `scikit-learn` and `numpy` was required?

Reply:
@adriangonz sorry about this wrong push. I have fixed the sklearn, numpy, xgb versions.